dev/com.ibm.ws.security.social/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -56,6 +56,12 @@ uniqueId.desc=The unique ID.
 authorizationEndpoint=Authorization end point URL
 authorizationEndpoint.desc=Specifies an Authorization end point URL.
 
+userInfoEndpoint=UserInfo end point URL
+userInfoEndpoint.desc=Specifies a UserInfo end point URL.
+
+userInfoEndpointEnabled=User Info endpoint enabled
+userInfoEndpointEnabled.desc=Specifies whether the User Info endpoint should be contacted.
```

Comment:
Avoid "should." Suggestion:
Specifies whether the User Info endpoint is contacted.